spec/requests/users_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET homepage" do
+    xit "acebook homepage" do
```

Comment:
This should be removed / if it is pending.
The sign in PR has the root url and this test working.
Might be worth removing this from here before merging into master.

Reply:
resolved